composer.json

```diff
@@ -31,7 +31,7 @@
         "symfony/options-resolver": "^2.3 || ^3.0"
     },
     "require-dev": {
-        "friendsofsymfony/rest-bundle": "^1.1",
+        "friendsofsymfony/rest-bundle": "<1.1",
```

Comment:
Yes it was added… would have been better to amend your commit and force push it, but it worked.